apps/web/src/components/digest-demo-flow/NodeStep.tsx

```diff
@@ -1,32 +1,41 @@
 import React, { useEffect, useState } from 'react';
 import styled from '@emotion/styled';
 import { Popover } from '@mantine/core';
+import { useLocation, useParams } from 'react-router-dom';
 
-import { useStyles } from '../../../../../design-system/template-button/TemplateButton.styles';
-import { colors, shadows, Text } from '../../../../../design-system';
-import { guidePreview, GuideTitleEnum, IBeat, IGuide } from './consts';
-import { useSegment } from '../../../../../components/providers/SegmentProvider';
-import { When } from '../../../../../components/utils/When';
+import { useStyles } from '../../design-system/template-button/TemplateButton.styles';
+import { colors, shadows, Text } from '../../design-system';
+import { guidePreview, guidePlayground, GuideTitleEnum, IBeat } from './consts';
+import { useSegment } from '../providers/SegmentProvider';
+import { When } from '../utils/When';
+import { ROUTES } from '../../constants/routes.enum';
+import { parseUrl } from '../../utils/routeUtils';
 
 export function NodeStep({
   data,
   id,
   Handlers,
   Icon,
   ActionItem,
+  ContentItem,
 }: {
-  data: any;
+  data: { label: string };
   id: string;
   Handlers: React.FC<any>;
   Icon: React.FC<any>;
-  ActionItem?: React.FC<any>;
+  ActionItem?: React.ReactNode;
+  ContentItem?: React.ReactNode;
 }) {
   const { theme } = useStyles();
   const { counter } = useCounter();
   const [sequence, setSequence] = useState<IBeat>();
+  const { pathname } = useLocation();
+  const { templateId = '' } = useParams<{ templateId: string }>();
+  const digestPlaygroundPathname = parseUrl(ROUTES.TEMPLATES_DIGEST_PLAYGROUND, { templateId });
+  const isDigestPlayground = pathname === digestPlaygroundPathname;
 
   const label = data.label.toLowerCase();
-  const popoverData = guidePreview[label] as IGuide;
+  const popoverData = isDigestPlayground ? guidePlayground[label] : guidePreview[label];
```

Comment:
based on the URL we use different texts for the popovers... I know this is not perfect implementation, but don't want to spend much more time on it to have a perfect, for now it's fine as this component will be only used in two places...